hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -550,13 +555,29 @@ public Cacheable getBlock(BlockCacheKey key, boolean caching, boolean repeat,
     return null;
   }
 
+  /**
+   * This method is invoked after the bucketEntry is removed from {@link BucketCache#backingMap}
+   * @param cacheKey
```

Comment:
Remove empty param

Reply:
@Apache9 , thank you very much for review. Fix it.